lib/Core/SymbolicError.cpp

```diff
@@ -251,7 +251,8 @@ SymbolicError::~SymbolicError() {
 }
 
 void SymbolicError::executeStore(ref<Expr> address, ref<Expr> value,
-                                 ref<Expr> error, ref<Expr> valueWithError) {
+                                 ref<Expr> error, ref<Expr> valueWithError,
+                                 KInstruction *i) {
```

Comment:
@Himeshi `KInstruction *i` can also be changed to `llvm::Instruction *instr` here.